### PR TITLE
TP-6352: Sticky column - back to top edge case bug

### PR DESCRIPTION
**Pull request by Guntrisoft**
- Sometimes when back to top is clicked, the browser gets
  confused, thinking that the sticky component is at the bottom.
- Added check to ensure that it returns to it's natural static
  position when back to top is pressed

### CODE REVIEW

File: app/assets/javascripts/components/StickyColumn.js [hunk 1]
```diff
@@ -148,13 +148,21 @@ define(['jquery', 'DoughBaseComponent', 'eventsWithPromises'], function($, Dough
       }
     }
 
-    if (this.isAtBottom && scrollTop < this.bottom) {
+    if (this.isAtBottom && scrollTop < this.bottom && scrollTop > this.top) {
```
Comment (benbarnett) [inline app/assets/javascripts/components/StickyColumn.js]: Do you think nesting these two `if (this.isAtBottom)` conditions in a single one would make it easier to read? I.e. how you've done it for `if (this.isFixed)` here https://github.com/moneyadviceservice/frontend/blob/sticky_column_back_top_fix/app/assets/javascripts/components/StickyColumn.js#L134

Comment (Guntrisoft) [inline app/assets/javascripts/components/StickyColumn.js]: fair - will change

**Comment by benbarnett** [comment]
:+1:

**Comment by jonalport** [comment]
Good job, Guntrisoft! :+1:

**Pull request merged**